src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFCommand.groovy

```diff
@@ -128,15 +129,16 @@ class LSFCommand extends SubmissionCommand {
      */
     @Override
     String assembleVariableExportString() {
-        List<String> environmentStrings = new LinkedList<>()
+        List<String> environmentStrings = []
 
-        if (passLocalEnvironment.isPresent()) {
-            if (passLocalEnvironment.get()) {
-                environmentStrings << "all"
-            } else {
-                environmentStrings << "none"
-            }
-        } // According to the docs (Version 10.1.0), undefined means "all".
+        if (passLocalEnvironment == PassVars.None) {
+            if (!parameters.isEmpty())
+                logger.warning("passLocalEnvironment is set to 'None' but you still request the passing of variables: ${parameters}")
```

Comment:
This will lead to a warning whenever a LSFCommand is assembled!

Reply:
O.k. right -- that can be quite a lot. I was considering an exception, because this condition corresponds to a misconfiguration. We'd had to change the client code then, but alas, for the sake of a clean interface :-)

Reply:
Indeed, now I think, a runtime-type exception (indicating a programming error) would be good. No client code change necessary.